src/main/java/jbse/algo/Algo_INVOKEMETA_Uninterpreted.java

```diff
@@ -91,6 +88,8 @@ private static boolean isSimple(State state, Reference objRef) throws FrozenStat
 		} else {
 			for (Variable var : obj.fields().values()) {
 				final Value val = var.getValue();
+				if (val instanceof Null)
+					return true;
```

Comment:
This is wrong: if a field is null the object is not simple. An object is simple if all its fields contain concrete values, or references to other objects that are (recursively) simple.